gis/whitebox-tools/Portfile

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        jblindsay whitebox-tools 1.2.0 v
+categories          gis
+platforms           darwin
+maintainers         nomaintainer
```

Comment:
would you consider becoming the maintainer of the port?

Reply:
OK, no worries